inc/util/any.h

```diff
@@ -396,6 +411,10 @@ const ValueType *any_cast(const Any *ptr) {
 
 inline void swap(Any &a1, Any &a2) noexcept { anyContainerSwap(a1, a2); }
 
+// Here the idea I'm suggesting above is more impractical, because
+//    the user is forced to explicitly specify the Policy template argument. 
+//    template<typename T, typename Policy, typename... Args>
+//    AnyContainer<Policy> make_any(Args &&... args) {
```

Comment:
Programming `make_AnyContainer` in the style you propose makes sense.  One question: how do you make a template with `T` which can't be deduced, `Policy` defaulted to the canonical policy, so that the user could use `make_any` as in the standard, but still deduce a trailing pack...
Overloading the name `make_any` for non-standard extensions is not 100% clearly good at my current understanding, for example, the extra template argument changes the arity of template arguments before the pack, if `make_any` is used in complicated templates, changing its arity may make otherwise perfectly valid code invalid.